src/sidebar/store/modules/annotations.ts

```diff
@@ -567,6 +573,37 @@ const savedAnnotations = createSelector(
   annotations => annotations.filter(ann => isSaved(ann)) as SavedAnnotation[],
 );
 
+/**
+ * Return the list of unique users who authored any annotation, ordered by username.
+ */
+const usersWhoAnnotated = createSelector(
+  (state: State) => state.annotations,
+  annotations => {
+    const usersMap = new Map<
+      string,
+      { user: string; username: string; displayName: string | null }
+    >();
+    annotations.forEach(anno => {
+      const { user } = anno;
+      const username = getUsername(user);
+      const displayName = anno.user_info?.display_name ?? null;
+
+      // Keep a unique list of users
+      if (!usersMap.has(user)) {
+        usersMap.set(user, { user, username, displayName });
+      }
+    });
+
+    // Sort users by username
+    return [...usersMap.values()].sort((a, b) => {
+      const lowerAUsername = a.username.toLowerCase();
+      const lowerBUsername = b.username.toLowerCase();
+
+      return lowerAUsername.localeCompare(lowerBUsername);
+    });
+  },
+);
```

Comment:
This returns a list of unique users ordered by username from the list of annotations.

It doesn't need to be a selector specifically, but since the annotations need to be fetched from the store, it was convenient.